src/generator.cs

```diff
@@ -732,6 +732,9 @@ public class DefaultValueFromArgumentAttribute : Attribute {
 public class NoDefaultValueAttribute : Attribute {
 }
 
+public class IgnoredInDelegateAttribute : Attribute {
```

Comment:
The name is fairly clear but please add a few comments where/why that attribute is being used (self-documentation).